src/openre.vcxproj

```diff
@@ -57,6 +57,9 @@
       <PreprocessorDefinitions>DEBUG;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
     </ClCompile>
+    <Link>
+      <AdditionalDependencies>winmm.lib;%(AdditionalDependencies)</AdditionalDependencies>
+    </Link>
```

Comment:
This is the same for debug and release, so you can move this up to be below line 52 instead and remove the current changes.